examples/nextjs-oauth/pages/api/authenticate.js

```diff
@@ -0,0 +1,54 @@
+import axios from "axios";
+import { serialize } from "cookie";
+
+export default function handler(req, res) {
+  let access_token = req.cookies.miro_access_token;
```

Comment:
Maybe good to mention in README that this is not best practice from security perspective as it will leak tokens to end users.